subprojects/docs/src/docs/userguide/publishingIvy.adoc

```diff
@@ -132,111 +86,57 @@ Overriding the default identity values is easy: simply specify the `organisation
-
-You can configure the tasks generated by the Ivy Publish Plugin to be skipped based on certain criteria. The following sample demonstrates how to restrict the `binary` publication to the `external` repository and the `binaryAndSources` publication to the `internal` repository.
-
-++++
-<sample dir="ivy-publish/conditional-publishing" id="publishingIvyConditionally" title="Configuring which artifacts should be published to which repositories">
-    <sourcefile file="build.gradle" snippet="task-config"/>
-    <output args="publish"/>
-</sample>
-++++
-
-Moreover, you may want to define your own shorthand tasks to fit your workflow. The following sample defines two tasks: `publishToExternalRepository` to publish all publications to the `external` repository and `publishToInternalRepository` for the `internal` repository:
-
-++++
-<sample dir="ivy-publish/conditional-publishing" id="shorthandTasks" title="Defining your own shorthand tasks for publishing">
-    <sourcefile file="build.gradle" snippet="shorthand-tasks"/>
-</sample>
-++++
+ * The name of the repository (optional)
+ * Its URL
+ 
+You can define multiple repositories as long as they have unique names within the build script. You may also declare one (and only one) repository without a name. That repository will take on an implicit name of "Ivy".
 
+You can also configure any authentication details that are required to connect to the repository. See api:org.gradle.api.artifacts.repositories.IvyArtifactRepository[] for more details.
 
 [[publishing_ivy:descriptor]]
 === Generating the Ivy module descriptor file without publishing
 
 At times it is useful to generate the Ivy module descriptor file (normally `ivy.xml`) without publishing your module to an Ivy repository. Since descriptor file generation is performed by a separate task, this is very easy to do.
 
-The Ivy Publish Plugin creates one api:org.gradle.api.publish.ivy.tasks.GenerateIvyDescriptor[] task for each registered api:org.gradle.api.publish.ivy.IvyPublication[], named `generateDescriptorFileFor«_PUBNAME_»Publication`, which will be `generateDescriptorFileForIvyJavaPublication` for the previous example of the `ivyJava` publication.
+The Ivy Publish Plugin creates one api:org.gradle.api.publish.ivy.tasks.GenerateIvyDescriptor[] task for each registered api:org.gradle.api.publish.ivy.IvyPublication[], named `generateDescriptorFileFor__PubName__Publication`, which will be `generateDescriptorFileForIvyJavaPublication` for the previous example of the `ivyJava` publication.
 
-You can specify where the generated Ivy file will be located by setting the `destination` property on the generated task. By default this file is written to `build/publications/«_PUBNAME_»/ivy.xml`.
+You can specify where the generated Ivy file will be located by setting the `destination` property on the generated task. By default this file is written to `build/publications/__PubName__/ivy.xml`.
```

Comment:
One could argue that about any of the conventions. Why is this one different? And if we remove it, what do we replace it with? If nothing, then aren't we just saying "tough, you're stuck with the defaults". We could easily say that we don't recommend changing the location and the example is there just as an example.

Reply:
This task produces a low-level intermediate result. What interest does the user have in it? And even if they are interested, why would they change the directory? It's just not a good use case, so I don't see why we would show it alongside other important concepts in the user manual.

Reply:
You say that it's an intermediate result, but it's the final result of explicitly running `generatePomFileFor__PubName__Publication`. I'm pretty sure I've run that myself in the past to verify the contents of the POM.

Anyway, I agree it's not the best example, but I think removing that example means removing the whole section. Which may be the best option. But I'm always loath to assume that users _won't_ want to do something, just because we can't think of a reason why they would. And when that happens, you end up with frustrated users and voodoo on StackOverflow.

I guess this will be tackled properly with lazy task configuration or some such. Or as part of that comprehensive overhaul of the docs you mentioned.

 - [ ] Remove this section

Reply:
I'm fine with mentioning this if you think seeing the POM is useful. I don't see why a user would want to customize it though. We can just point them at the default output if they want to have a peek.

Reply:
The task and its behaviour are covered in the plugin chapter. The main reason for it being here is as an example of customising the publishing tasks. If we don't have the example, then the section can go. I'm fine with that.